website/pages/docs/core-concepts/configuration.md

```diff
@@ -10,7 +10,7 @@ One option is to maintain configuration for your source and destination plugins
 
 Here is a simple example with only one source and one destination plugin:
 
-```yaml title="aws.yml"
```

Comment:
FYI the `copy` was removed intentionally for code blocks that have the versions placeholder due to https://github.com/cloudquery/cloudquery/issues/5486.

However I tested it after the upgrade to the newest Nextra version and its seems to be working now so all good 😄 I was planning to re-add those today, so you saved me some work 🙌

Reply:
Ah, I forgot about that, but glad it's working now! 😄